testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jsf/unsupportedwar/web.xml

```diff
@@ -15,16 +15,16 @@
     limitations under the License.
 -->
 <web-app version="4.0"
-    xmlns="http://xmlns.jcp.org/xml/ns/javaee"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_4_0.xsd">
+         xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_4_0.xsd">
```

Comment:
This should be version 6 (line 17) and the jakarta namespace. Still a draft: https://jakarta.ee/xml/ns/jakartaee/#10

Reply:
Noted, but I wouldn't spend too much on this just yet, as this is very much a work in progress. I just created the draft so I could easily see the changes in one place. :)

Reply:
Consider it NIT :-) And *I* learn from 99% of the rest ;-)